tests/probes/rpm/rpm_common.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2015 Red Hat Inc., Durham, North Carolina.
+# All Rights Reserved.
+#
+# OpenScap Probes Test Suite.
+
+RPMBASE="${builddir}/tests/probes/rpm"
+RPMTEST="${RPMBASE}/root"
+RPMBUILD="${RPMBASE}/build"
+
+
+function rpm_build {
+    require "rpmbuild" || return 255
+    if [ ! -f ${RPMBUILD}/RPMS/noarch/foo-1.0-1.noarch.rpm ]; then
+        rpmbuild --define "_buildhost testhost" --define "_tmppath ${RPMTEST}/tmp" --define "_topdir ${RPMBUILD}" -ba ${srcdir}/../foo.spec
```

Comment:
We will need to add "rpm-build" to the list of test dependencies in Developer Guide, section 1.5 https://github.com/OpenSCAP/openscap/blob/maint-1.3/docs/developer/developer.adoc